### ISSUE TITLE
Menu items and context menus invisible

### ISSUE THREAD

**Issue by username227** [transmission-remote-gui/transgui#1410]
Hi all, I recently installed the latest version into Linux mint. When I click on the menu in the top, the drop down is mostly invisible. Same with the context menus from a right click on a torrent.

If I switch to a dark theme, I can read them. But I don't want to use a dark theme system wide. How can I get the text in these menus to be black do they'll be visible? Thanks.

**Comment by PeterDaveHello** [comment]
@username227 do you have any screenshot that we can take a look?

**Comment by username227** [comment]
Yes, here they are. Sorry, I would've done with the original post but I just got home. Also note that at the bottom, the column headers are dark. Not sure why that is either.

![2022-02-08-145350_1920x1080_scrot](https://user-images.githubusercontent.com/88214024/153065428-733b9910-2830-4697-93df-51d4baf86790.png)
![2022-02-08-145520_1920x1080_scrot](https://user-images.githubusercontent.com/88214024/153065432-7fcf567e-4f0d-4c72-9d7d-4ec6474acc44.png)

**Comment by GRMrGecko** [comment]
My fix for right now is to force the standard Adwaita theme:

```
[Desktop Entry]
Name=Transmission Remote GUI
Comment=Remote transmission control
Exec=env GTK_DATA_PREFIX=/usr/share/themes/Adwaita /opt/src/transgui-5.18.0/transgui
Icon=/opt/src/transgui-5.18.0/transgui.png
Terminal=false
Type=Application
Categories=Utility;Network;
```

Hope this helps
